.github/workflows/ci.yml

```diff
@@ -47,6 +47,11 @@ jobs:
             rust-target: aarch64-unknown-linux-musl
             binary: fluvio
             run: make build-cli
+          - os: ubuntu-latest
+            rust: stable
+            rust-target: arm-unknown-linux-gnueabihf
+            binary: fluvio
+            run: make build-cli
```

Comment:
Remove run. Should not have any impl in matrix

Reply:
This was already like this, if we want to change it we should do it in another PR, I don't want to rewrite a CI flow here.

Reply:
Then dont add to it